bot/exts/evergreen/issues.py

```diff
@@ -17,40 +17,57 @@
     Tokens,
     WHITELISTED_CHANNELS
 )
+from bot.utils.decorators import whitelist_override
+from bot.utils.extensions import invoke_help_command
 
 log = logging.getLogger(__name__)
 
 BAD_RESPONSE = {
     404: "Issue/pull request not located! Please enter a valid number!",
     403: "Rate limit has been hit! Please try again later!"
 }
+REQUEST_HEADERS = {
+    "Accept": "application/vnd.github.v3+json"
+}
 
-MAX_REQUESTS = 10
-REQUEST_HEADERS = dict()
+REPOSITORY_ENDPOINT = "https://api.github.com/orgs/{org}/repos?per_page=100"
```

Comment:
Do we want to specify the `type=public` query param here to ensure we never pull private repos? Just in case the token given to lancebot gets given read access to them.

Reply:
Added it in 51d9b54 as a safety measure